Modules/FIT/FT0/etc/ft0-post-processing.json

```diff
@@ -23,6 +23,154 @@
       }
     },
     "checks": {
+      "ASideInnerMIPCheck": {
+        "active": "true",
+        "className": "o2::quality_control_modules::fit::MIPCheck",
+        "moduleName": "QcFT0",
+        "policy": "OnAny",
+        "detectorName": "FT0",
+        "dataSource": [
+          {
+            "type": "PostProcessing",
+            "name": "PostProc",
+            "MOs": [
+              "AmpAInner"
+            ]
+          }
+        ],
+        "checkParameters": {
+          "nameObjectToCheck": "AmpAInner",
+          "nPeaksToFit": "2",
+          "gausParamsMean": "14, 28",
+          "gausParamsSigma": "3, 7",
+          "fitRangeLow": "11",
+          "fitRangeHigh": "35",
+          "meanWarningsLow": "12, 26",
+          "meanWarningsHigh": "16, 30",
+          "meanErrorsLow": "10, 24",
+          "meanErrorsHigh": "18, 32",
+          "sigmaWarnings": "4, -1",
+          "sigmaErrors": "5, -1",
+          "drawMeanWarningsLow": "false, false",
+          "drawMeanWarningsHigh": "false, false",
+          "drawMeanErrorsLow": "true, true",
+          "drawMeanErrorsHigh": "true, true",
+          "drawSigmaWarnings": "false, false",
+          "drawSigmaErrors": "fasle, false",
```

Comment:
I'm not sure but maybe it is better to use and parse list of json entities, smth like:
```
"peaks": [
  {
     "meanWarningsLow": 12,
     "meanWarningsHigh": 16,
     ....
      "draw_options": ["drawMeanErrorsLow", "drawSigmaWarnings"]
  },
  {
      ...
   }
]
```
and parse it via boost property tree or `RapidJSON`. Then no need in vec of bools, you will just need to enumerate list of options

Reply:
Yes that might be a good idea, let me think about it for the next iteration. One benefit (maybe?) of the current approach is that one can tell what the options are, just by looking at the config, and not having to jump over to the documentation...